controllers/istio/config.go

```diff
@@ -0,0 +1,9 @@
+package istio
+
+import (
+	"github.com/Dynatrace/dynatrace-operator/logger"
+)
+
+var (
+	log = logger.NewDTLogger().WithName("dynakube-istio")
```

Comment:
isn't `istio` enough as name? or `istio-controller` to fit the other controllers

Reply:
its not a controller its just named that :sweat_smile: 
the `dynakube` part is there because its called from the `dynakube-controller`

Reply:
makes sense, not really redundant then